docs/reference/environment-variables.rst

```diff
@@ -0,0 +1,101 @@
+*********************
+Environment Variables
+*********************
+
+Applications built on craft-application have several environment variables that
+can configure their behaviour. They and the behaviour they modify are listed
+below.
+
+Variables passed to managed builders
+------------------------------------
+
+Several environment variables from the host environment are passed to the
+managed build environment. While an application may adjust these by adjusting
+the ``environment`` dictionary attached to the ``ProviderService``,
+craft-application will by default forward the ``http_proxy``, ``https_proxy``
+and ``no_proxy`` environment variables from the host.
+
+Supported variables
+-------------------
+
+These variables are explicitly supported for user configuration.
+
+.. _env-var-craft-build-environment:
+
+``CRAFT_BUILD_ENVIRONMENT``
+===========================
+
+Allows an environment to tell a craft application to run directly on the host
+rather than in managed mode if the value is ``host``. This is roughly equivalent
+to using ``--destructive-mode``, but is designed for configurations where the
+application is already being run in an appropriate container or VM, such as
+`Snapcraft rocks <https://github.com/canonical/snapcraft-rocks/>`_ or
+when controlled by a CI system such as `Launchpad <https://launchpad.net>`_.
+
+``CRAFT_BUILD_FOR``
+===================
+
+Sets the default architecture to build for. Is overridden by ``--build-for`` in
```

Comment:
Consider:

```suggestion
Sets the default architecture to build for. Overridden by ``--build-for`` in
```

We can slightly soften the passive by removing the stative "is". It further helps skimmability.